### PR TITLE
Bugfix/allow profile visibility

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: lib/profiles/user-profile-data.js [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+
+    function filterFields (err, profile, requestingUserProfile) {
+      var allowedFields = [];
+
+      if (_.contains(profile.userTypes, 'attendee-o13')) {
+        allowedFields = _.union(allowedFields, allowedOptionalFields['attendee-o13']);
+      }
+
+      if (_.contains(profile.userTypes, 'champion')) {
+        allowedFields = _.union(allowedFields, allowedOptionalFields['champion']);
```
Comment (DanielBrierton) [inline lib/profiles/user-profile-data.js]: @rosalanghammer If we have an attendee-o13 who's also a champion (Youth Champion), should their profile be as open as an adult champion, or still restricted as an attendee-o13?

File: scripts/database/pg/migrations/021.do.force-private-true.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+DO $$
+    BEGIN
+        BEGIN
+          UPDATE cd_profiles SET private = true WHERE private IS NULL;
```
Comment (DanielBrierton) [inline scripts/database/pg/migrations/021.do.force-private-true.sql]: Should also alter table to make default true for all new entries? I know it's done in the µs, but doing it in the DB schema seems a little cleaner IMO.

File: service.js [hunk 3]
```diff
@@ -79,18 +69,17 @@ require('./migrate-psql-db.js')(function (err) {
   process.on('SIGTERM', shutdown);
   process.on('uncaughtException', shutdown);
 
-  function shutdown (err) {
```
Comment (DanielBrierton) [inline service.js]: 🙄

File: scripts/database/pg/migrations/021.do.force-private-true.sql (hunk 2 above)
Comment (Wardormeur) [inline scripts/database/pg/migrations/021.do.force-private-true.sql]: > not null in db

**Pull request merged**